### PR TITLE
Add placeholderSrc if AMP

### PR DESCRIPTION
**Pull request by pjlee11**
part of 4044

Requires the image in https://github.com/bbc/simorgh/pull/4048 to be publicly available 

**Overall change:** If the request is AMP add `placeholderSrc` if it's canonical add `showPlaceholder=false`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

Testing ensure a placeholder loads on `.amp` and the validation warning has been removed.

### CODE REVIEW

File: src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx [hunk 1]
```diff
@@ -11,6 +11,9 @@ import { RequestContext } from '#contexts/RequestContext';
 import { ServiceContext } from '#contexts/ServiceContext';
 import embedUrl from '../../../MediaPlayer/helpers/embedUrl';
 
+const liveRadioPlaceholderImageSrc =
+  'https://news.files.bbci.co.uk/include/articles/public/images/audio-player-placeholder.png';
```
Comment (chris-hinds) [inline src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx:15]: Potentially a stupid question but should we be putting this image through iChef to ensure the most optimal size is returned?

Comment (pjlee11) [reply src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx:15]: For it to go through ichef it needs to be published on one of the stacks ichef knows about. Given we want this fixed quickly just using the single static image and allowing the browser to stretch it should be fine. 

We can look to improve this when UX provide us an asset (this image is currently one I've generated temporarily), and then we can evaluate what this placeholder for AMP should look like on mobile widths.

**Review by chris-hinds — APPROVED** [review]

**Comment by pjlee11** [comment]
MP4 example of the AMP placeholder being loaded before the SMP iframe is loading into the page.

[Audio placeholder for AMP.mp4.zip](https://github.com/bbc/simorgh/files/3675776/Audio.placeholder.for.AMP.mp4.zip)
Had to zip the MP4 😞

File: package.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
     "@bbc/psammead-inline-link": "^1.3.9",
     "@bbc/psammead-locales": "^2.19.0",
     "@bbc/psammead-media-indicator": "^2.6.2",
-    "@bbc/psammead-media-player": "^1.0.1-alpha.3",
+    "@bbc/psammead-media-player": "^1.1.1-alpha.2",
```
Comment (pjlee11) [inline package.json:71]: Working on getting this to 2.0.0 in https://github.com/bbc/psammead/pull/2280

Comment (pjlee11) [inline package.json]:
```suggestion
    "@bbc/psammead-media-player": "^1.1.1-alpha.3",
```

File: src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx [hunk 3]
```diff
@@ -11,6 +11,9 @@ import { RequestContext } from '#contexts/RequestContext';
 import { ServiceContext } from '#contexts/ServiceContext';
 import embedUrl from '../../../MediaPlayer/helpers/embedUrl';
 
+const liveRadioPlaceholderImageSrc =
+  'https://news.test.files.bbci.co.uk/include/articles/public/images/audio-player-placeholder.png';
```
Comment (pjlee11) [inline src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx]:
```suggestion
  'https://news.files.bbci.co.uk/include/articles/public/images/audio-player-placeholder.png';
```

File: package.json (hunk 2 above)
Comment (ryanmccombe) [inline package.json]: This is a weird bump

**Review by ryanmccombe — APPROVED** [review]

**Review by ibMadbouly — APPROVED** [review]
👍

**Pull request merged**